src/service/controllers/auth.js

```diff
@@ -74,30 +75,54 @@ exports.pcToken = (req, res) => {
 				code: 0
 			},
 			data: {
-				access_token: access_token,
+				access_token,
 				expires_in: 1800
 			}
 		};
 		res.send(JSON.stringify(data));
 }
 
+exports.jwToken = (req, res) => {
+  const token = jwt.sign({ username: userByCode(req) }, 'secret');
+  res.set({ 'Content-type': 'application/json' });
+  var data = {
+    status: {
+      code: 0
+    },
+    data: {
+      access_token: token,
+      expires_in: 1800
+    }
+  };
+  res.send(JSON.stringify(data));
+}
```

Comment:
We now export a real token using the **jsonwebtoken** third party, because it is usually decoded in the plugin service.